<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-NHMM4IQC" failures="1" tests="1" name="Test" time="0.066" errors="0" timestamp="2020-06-12T20:57:55 BST">
  <testcase classname="com.vcit.automation.TestCheckOrder" name="testCheckOrder" time="0.011">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.vcit.automation.TestCheckOrder.testCheckOrder(TestCheckOrder.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testCheckOrder -->
  <testcase classname="com.vcit.automation.TestLogin" name="testCustomerRegistrationWithValidEmail" time="0.0">
    <ignored/>
  </testcase> <!-- testCustomerRegistrationWithValidEmail -->
  <testcase classname="com.vcit.automation.TestLogin" name="testCustomerRegistrationWithInvalidEmail" time="0.0">
    <ignored/>
  </testcase> <!-- testCustomerRegistrationWithInvalidEmail -->
</testsuite> <!-- Test -->
